### DISCUSSION TITLE
Still calculating the usage of a deleted project

### DISCUSSION THREAD

**Discussion by ahmedelhalwagy3** [supabase/supabase#26824] [Questions]
I had 2 projects in my organization, however one was deleted a while ago, but supabase still acknowledges the data from this project and is calculated in the usage section besides my other project. Any solutions?

**Comment by GaryAustin1** [comment]
Is a while ago over 30days?

**Comment by ahmedelhalwagy3** [comment]
No, but I already deleted some edge functions after they exceeded the limit
, am i going to be billed upon the exceeded limit? Write now I have only 9
edge functions.

**Reply by GaryAustin1** [reply]
Are you on a free plan?  If so there is no billing, just limits that might impact use of your instance.

**Comment by ahmedelhalwagy3** [comment]
I am on free plan.

The impact you are talking about might be the db will become unresponsive.

My question is am I going to accounted for the 11 or the 9

Note: I had 11 edge functions then I deleted 2 the next day when it said i
exceeded the limit.

**Reply by GaryAustin1** [reply]
You should normally reply the "answer" and not generate a new answer each time.

The edge functions are a limit in time I'm pretty sure.  If you drop to get below the limit you should be fine.  Same with database size.  If you exceed and then reduce it then your system would not stay in read only mode.

Exceeding egress is a different story as there is no way to "take it back".

**Comment by ahmedelhalwagy3** [comment]
Ok, thanks for the clarification.
